crates/iota-graphql-rpc/src/main.rs

```diff
@@ -4,37 +4,30 @@
 
 use std::{fs, path::PathBuf};
 
-use clap::Parser;
+use clap::{CommandFactory, FromArgMatches};
 use iota_graphql_rpc::{
     commands::Command,
     config::{ConnectionConfig, Ide, ServerConfig, ServiceConfig, TxExecFullNodeConfig, Version},
     server::{builder::export_schema, graphiql_server::start_graphiql_server},
 };
 use tokio_util::{sync::CancellationToken, task::TaskTracker};
 
-// Define the `GIT_REVISION` const
-bin_version::git_revision!();
+// Define the `GIT_REVISION` and `VERSION` consts
+bin_version::bin_version!();
 
 // VERSION mimics what other iota binaries use for the same const
-static VERSION: Version = Version {
+static VERSION_VAL: Version = Version {
```

Comment:
should this comment now say `VERSION_VAL` mimics .... ?

Reply:
Yeah, probably 😆